libr/core/cmd_debug.c

```diff
@@ -3924,14 +3924,14 @@ static int cmd_debug(void *data, const char *input) {
 				break;
 			case 't':
 				if (input[3] == ' ') {
-					r_debug_session_save (core->dbg, input + 4);
+					r_debug_session_save (core->dbg, core->config, input + 4);
```

Comment:
rdebug cant handle core->config, dont pass that pointer. keep the old session-save signature and do the following changes:

* add new api r_debug_session_path()
* call it before calling r_debug_session_save (or when changing config var by adding a callback)